save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/config/ConfigProperties.kt

```diff
@@ -56,6 +58,17 @@ data class DockerSettings(
     val runtime: String = "runc",
 )
 
+/**
+ * @property apiServerUrl see https://kubernetes.io/docs/tasks/run-application/access-api-from-pod/
```

Comment:
could you please add extra info here - very difficult to understand what are these properties for

Reply:
Added